app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from __future__ import annotations
```

Comment:
do you need this? 
If yes, i guess this is a good time to update at least to 3.10 :))
but it is up to you, ofcourse

Reply:
Fixed